overlord/state/export_test.go

```diff
@@ -32,4 +32,8 @@ func ChangeUnlockCheckpointRetryParamsForTest(newInterval, newMaxTime time.Durat
 	return
 }
 
+func (r *TaskRunner) Handlers() map[string]HandlerFunc {
```

Comment:
I have no strong opinion either way, I slightly lean towards just making it available on TaskRunner, but happy to move back to export_test.go with a less confusing name.